### DISCUSSION TITLE
Content Secure Policy

### DISCUSSION THREAD

**Discussion by nhatdn** [mozilla/hubs#5646] [Hubs Cloud Questions]
![image](https://user-images.githubusercontent.com/85542339/183397722-4c986667-6540-43b9-ab5c-e19abcbd5261.png)
 
I have a problem that I need to help, we are getting the error as below but there isn't solution. Please can someone help me, Thanks all.

**Comment by pattersonbl2** [comment]
it looks like an inline scprit added in the admin panel needs to be removed. It would help if you got to your admin panel and removed it from the server setup.

**Reply by nhatdn** [reply]
i deleted all from server setup but it has error in tab console
